product_video/models/media_video_provider.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MediaVideoProvider(models.Model):
+    _name = "media.video.provider"
+    _description = "Media Video Provider"
+
+    name = fields.Char()
+    code = fields.Char()
```

Comment:
what is the interest of the "code" field?

Reply:
It's a technical name, for example if you can use it in your front end (shopinavder, odoo, or whatever) to use the right wigdet

Reply:
Why not a sélection better ? YouTube / vimeo / peertube / etc ?
It could also allow in a second time to add an onchange on url video to have the possibility to get the name and the thumbnail of the video. Depending on the technology.
(Not a blocking point)

Reply:
the idea is be able to create new provider without customising the code, so all provider can be added with data